analyzers/src/SonarAnalyzer.Common/Rules/MultipleVariableDeclarationBase.cs

```diff
@@ -34,11 +28,10 @@ public abstract class MultipleVariableDeclarationBase : SonarDiagnosticAnalyzer
         protected abstract GeneratedCodeRecognizer GeneratedCodeRecognizer { get; }
     }
 
-    public abstract class MultipleVariableDeclarationBase<TLanguageKindEnum,
-        TFieldDeclarationSyntax, TLocalDeclarationSyntax> : MultipleVariableDeclarationBase
+    public abstract class MultipleVariableDeclarationBase<TLanguageKindEnum, TFieldDeclarationSyntax, TLocalDeclarationSyntax> : MultipleVariableDeclarationBase
```

Comment:
When you're touching this, it would be great to really clean it up - remove the nongeneric base and inherit `SonarDiagnosticAnalyzer<TSyntaxKind>` to remove ton of useless scaffolding from all 3 classes